src/config.c

```diff
@@ -27,7 +27,7 @@ void config_load()
         int c;
         char s[256];
         char *p;
-        sprintf(s, "%sb-em.cfg", exedir);
+        snprintf(s, sizeof(s), "%sb-em.cfg", exedir);
```

Comment:
Barely, since you're not checking the return value.  Better would be a wrapper around this, but don't bother.  I just want to point out this change is no better than leaving it as it is.